compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -5268,4 +5270,107 @@ private void visitBinaryLogicalExpr(BLangBinaryExpr binaryExpr) {
 
         result = rewriteExpr(stmtExpr);
     }
+
+    /**
+     * Split packahe init function into several smaller functions.
+     *
+     * @param packageNode package node
+     * @param env symbol environment
+     * @return initial init function but trimmed in size
+     */
+    private BLangFunction splitFunction(BLangPackage packageNode, SymbolEnv env) {
```

Comment:
Shall we rename the method name to splitInitFunction? Also there is a typo in doc comment "Split packahe"